### PR TITLE
Add missing types in code templates

### PR DESCRIPTION
**Pull request by nicolas-grekas**

### CODE REVIEW

File: src/Resources/skeleton/doctrine/Repository.tpl.php [hunk 1]
```diff
@@ -70,33 +70,29 @@ public function upgradePassword(<?= sprintf('%s ', $password_upgrade_user_interf
...
+//    {
+//        return $this->createQueryBuilder('<?= $entity_alias; ?>')
+//            ->andWhere('<?= $entity_alias; ?>.exampleField = :val')
+//            ->setParameter('val', $value)
+//            ->orderBy('<?= $entity_alias; ?>.id', 'ASC')
+//            ->setMaxResults(10)
+//            ->getQuery()
+//            ->getResult()
+//        ;
+//    }
```
Comment (nicolas-grekas) [inline src/Resources/skeleton/doctrine/Repository.tpl.php:86]: sticking to a consistent CS for examples

File: src/Resources/skeleton/test/ApiTestCase.tpl.php [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 class <?= $class_name ?> extends ApiTestCase
 {
-    public function testSomething(): void
+    public function testSomething()
```
[the remark this answers is not in the document]
Comment (jrushlow) [reply src/Resources/skeleton/test/ApiTestCase.tpl.php:9]: I think this is more of an IDE thing. E.g. PHPStorm by default flags methods w/o return types. 
image

Granted, they're not needed - but they're also not hurting anything... Thoughts?

**Review by jrushlow — COMMENTED** [review]
Looks good! Thanks @nicolas-grekas quick comment on the test return types.

/cc @weaverryan

File: src/Resources/skeleton/test/ApiTestCase.tpl.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (nicolas-grekas) [reply src/Resources/skeleton/test/ApiTestCase.tpl.php:9]: :facepalm:

[the remark this answers is not in the document]
Comment (nicolas-grekas) [reply src/Resources/skeleton/test/ApiTestCase.tpl.php]: reverted

**Review by weaverryan — APPROVED** [review]
This is really nice 👍

**Review by OskarStark — APPROVED** [review]

**Comment by weaverryan** [comment]
Thanks Nicolas!

**Pull request merged**